settings.gradle.kts

```diff
@@ -48,6 +42,9 @@ rootProject.name = "govuk-mobile-android-app"
 
 enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
 include(":app")
+include(":analytics")
```

Comment:
Whilst we're cleaning up these imports here perhaps put them all in alphabetical order as likely to grow to a large list in the future?